modules/cluster-logging-collector-fluentd.adoc

```diff
@@ -7,11 +7,6 @@
 
 You can use the Fluentd *forward* plug-ins to send a copy of your logs to an external log aggregator, instead of the default Elasticsearch.
```

Comment:
You can use the Fluentd *forward* plug-ins to send your logs to both the internal Elasticsearch instance and an outside log aggregator